examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
 # callback which prints training loss and perplexity once in a while
 if not args.only_mlm_loss:
     log_tensors = [train_loss, mlm_loss, nsp_loss]
+    eval_log_tensors = [eval_loss, eval_mlm_loss, eval_nsp_loss]
```

Comment:
Do you use all these tensors in the eval callbacks?

Reply:
yes